lib/storage/src/content_manager/mod.rs

```diff
@@ -40,7 +40,7 @@ pub mod consensus_ops {
         },
         RemovePeer(PeerId),
         RequestSnapshot {
-            request_index: Option<u64>,
+            request_index: Option<u64>, // TODO: param is not used, kept for compatibility
         },
```

Comment:
This message only ever send locally (e.g., from the node to itself, to propagate the message to consensus thread), so I've removed the `request_index` field.